src/main/java/org/apache/solr/benchmarks/BenchmarksMain.java

```diff
@@ -105,32 +108,124 @@ public static void runQueryBenchmarks(List<QueryBenchmark> queryBenchmarks, Stri
 		    for (int threads = benchmark.minThreads; threads <= benchmark.maxThreads; threads++) {
 		        QueryGenerator queryGenerator = new QueryGenerator(benchmark);
 		        HttpSolrClient client = new HttpSolrClient.Builder(baseUrl).build();
+		        String collection = collectionNameOverride==null? benchmark.collection: collectionNameOverride;
 		        ControlledExecutor controlledExecutor = new ControlledExecutor(threads,
 		                benchmark.duration,
 		                benchmark.rpm,
 		                benchmark.totalCount,
 		                benchmark.warmCount,
-		                getQuerySupplier(queryGenerator, client, collectionNameOverride==null? benchmark.collection: collectionNameOverride));
+		                getQuerySupplier(queryGenerator, client, collection));
 		        long start = System.currentTimeMillis();
+						List<SolrBenchQuery> generatedQueries;
 		        try {
-		            controlledExecutor.run();
+		            generatedQueries = controlledExecutor.run();
 		        } finally {
 		            client.close();
 		        }
 
+	        	int success = 0, failure = 0;
+
+		        if (StressMain.generateValidations) {
+		        	int totalDocsIndexed = getTotalDocsIndexed(baseUrl, collection);
+		        			
+		        	FileWriter outFile = new FileWriter("suites/validations-"+benchmark.queryFile+"-docs-"+totalDocsIndexed+"-queries-"+benchmark.totalCount+".tsv");
```

Comment:
Shall we use json for this file since:
1. The facet part is already formatted as json
2. This would give us better flexibility to do different types of comparison that TSV. For example we can add the total doc indexed as a part of the result and also verify different params per query result in the future (more than just numFound and facets)

Reply:
+1 to json.